math/maxima/Portfile

```diff
@@ -3,46 +3,73 @@
 PortSystem 1.0
 
 name            maxima
-version         5.39.0
-revision        14
+version         5.41.0
 categories      math
 maintainers     mareimbrium.org:kuba openmaintainer
 platforms       darwin
 license         GPL-2+
 description     The Maxima computer algebra system
-long_description \
-        Maxima is a system for the manipulation of symbolic \
-        and numerical expressions, including differentiation, \
-        integration, Taylor series, Laplace transforms, ordinary \
-        differential equations, systems of linear equations, \
-        polynomials, and sets, lists, vectors, matrices, and \
-        tensors. Maxima yields high precision numeric results \
-        by using exact fractions, arbitrary precision integers, \
-        and variable precision floating point numbers. Maxima \
-        can plot functions and data in two and three dimensions.
-
 homepage        http://maxima.sourceforge.net/
-master_sites    sourceforge
 
-checksums       md5     3bccafb512ff1054be700c61601088af \
-                sha256  f58ff98013fcaa37d8661b941672c38db58f982f082d6cfcca1f1b5fac7d71b3 \
-                rmd160  3c4b153554c9b9cb3442e030d503fbe975a614fc
+subport maxima-devel {
+    conflicts   maxima
+    long_description ${description}: \n\
+                Provides the development version, which is typically updated every week.
+
+    # git describe --tags : branch-5_41-base-415-g7e71fc4
+    version     5.41post
+    fetch.type  git
+    git.url     https://git.code.sf.net/p/maxima/code
+    git.branch  7e71fc4b6a95cab6418227754f01fc44ac845da1
+
+    post-fetch {
+        system -W ${worksrcpath} "git describe --tags"
```

Comment:
Why adding a `git describe` command? Users won't see this if they installed the port from binary archives or didn't use `-v` or `-d` with `port install`. We should be able to infer the specific commit from port's version and revision.

Reply:
I will remove the 'git describe --tags' line.